ts/a11y/explorer/ExplorerPool.ts

```diff
@@ -295,6 +297,16 @@ export class ExplorerPool {
     this.highlighter.unhighlight();
   }
 
+  /**
+   * Convenience method to return the speech explorer of the pool with the
+   * correct type.
+   *
+   * @return {SpeechExplorer}
+   */
+  public speech(): SpeechExplorer {
```

Comment:
Would this be better as a getter so you don't need the `()` every time?  I always go back and forth about that.

Reply:
Done.